src/link/script.c

```diff
@@ -339,10 +341,11 @@ static void processCommand(enum LinkerScriptCommand command, uint16_t arg, uint1
 		break;
 
 	case COMMAND_ALIGN:
-		if (arg >= 16)
-			arg = 0;
-		else
-			arg = (*pc + (1 << arg) - 1) & ~((1 << arg) - 1);
+		arg = arg >= 16 ? 0 : (*pc + (1 << arg) - 1) & ~((1 << arg) - 1);
```

Comment:
I liked better when the conditions were split apart... this should at least be split into several lines, imo.

Reply:
Okay. In that case, may as well factor out the repeated subexpression too.